net/tcp/Kconfig

```diff
@@ -93,7 +93,7 @@ config NET_TCP_MAX_CONNS
 
 config NET_TCP_NPOLLWAITERS
 	int "Number of TCP poll waiters"
-	default 1
+	default 2
```

Comment:
why?

Reply:
since most drivers set the default value to 2, it's better to align with others.